content/webapp/pages/search/works.tsx

```diff
@@ -57,7 +57,7 @@ const SortPaginationWrapper = styled.div`
   flex-wrap: wrap;
 `;
 
-export const CatalogueSearchPage: NextPageWithLayout<Props> = ({
+const CatalogueSearchPage: NextPageWithLayout<Props> = ({
```

Comment:
No need to export here, it's exported at the bottom of the file in line with the other pages.